DESCRIPTION

```diff
@@ -1,20 +1,27 @@
 Package: bluedotmapper
-Title: What the Package Does (One Line, Title Case)
+Title: Generate georeferenced PDFs
 Version: 0.0.0.9000
-Authors@R: 
-    c(person(given = "First",
-             family = "Last",
-             role = c("aut", "cre"),
-             email = "first.last@example.com"),
-      person(given = "Province of British Columbia",
-             role = "cph"))
-Description: What the package does (one paragraph).
+Authors@R: as.person(c(
+    "Andy Teucher <andy.teucher@gov.bc.ca> [aut, cre]",
+    "Gail Roth <gail.roth@gov.bc.ca> [aut, cre]",
```

Comment:
the `cre` tag is only used for package maintainer, just `[aut]` is good for everyone else